docs/topics/i18n/translation.txt

```diff
@@ -959,15 +959,77 @@ Django provides an integrated solution for these problems: It passes the
 translations into JavaScript, so you can call ``gettext``, etc., from within
 JavaScript.
 
+The main solution to these problems is the following ``JavaScriptCatalog`` view,
+which sends out a JavaScript code library with functions that mimic the
+``gettext`` interface, plus an array of translation strings.
+
 .. _javascript_catalog-view:
 
-The ``javascript_catalog`` view
--------------------------------
+The ``JavaScriptCatalog`` view
+------------------------------
 
 .. module:: django.views.i18n
 
+.. versionadded:: 1.10
+
+.. class:: JavaScriptCatalog
+
+    A view that produces a JavaScript code library with functions that mimic
+    the ``gettext`` interface, plus an array of translation strings.
+
+    **Attributes**
+
+    .. attribute:: domain
+
+        Translation domain containing strings to add in the view output.
+        Defaults to ``'djangojs'``.
+
+    .. attribute:: packages
+
+        A list of app labels among installed applications in the Python
```

Comment:
I think "app labels" isn't a good term to use because it could be confused with https://docs.djangoproject.com/en/1.9/ref/models/options/#app-label.

suggestion: A list of dotted Python paths to packages (typically Django applications in INSTALLED_APPS) containing a locale directory.